cargo-dist/src/main.rs

```diff
@@ -516,3 +519,57 @@ fn cmd_manifest_schema(
     }
     Ok(())
 }
+
+async fn cmd_update(_config: &Cli, args: &cli::UpdateArgs) -> Result<(), miette::ErrReport> {
+    let mut updater = AxoUpdater::new_for("cargo-dist");
+    // Do we want to treat this as an error?
+    // Or do we want to sniff if this was a Homebrew installation?
+    if updater.load_receipt().is_err() {
+        eprintln!("Unable to load install receipt to check for updates.");
+        eprintln!("If you installed this via `brew`, please `brew upgrade cargo-dist`!");
+        return Ok(());
+    }
+
+    if !updater.check_receipt_is_for_this_executable()? {
+        eprintln!("This installation of cargo-dist wasn't installed via a method that `cargo dist update` supports.");
+        eprintln!("Please upgrade manually.");
+        return Ok(());
+    }
+
+    if let Some(result) = updater.run().await? {
+        eprintln!(
+            "Update performed: {} => {}",
+            env!("CARGO_PKG_VERSION"),
+            result.new_version
+        );
+
+        // At this point, we've either updated or bailed out;
+        // we can proceed with the init if the user would like us to.
+        if !args.skip_init {
```

Comment:
i think it might be useful to print something to the screen so the user knows what's happening